### ISSUE TITLE
Stability regions

### ISSUE THREAD

**Issue by marknovak** [ingemanKE/IGP-adaptive-defense#9]
Need to ensure (using `Manipulate`) that unstable regions actually do denote cycling dynamics.  (So far this has only been done for Fig. 3 with which the stability analysis code was developed.)

**Comment by ingemanKE** [comment]
What would the alternative be in unstable regions...chaotic dynamics? Do we need to differentiate those to satisfy reviewer commment?

**Comment by marknovak** [comment]
It's possible that in the other models and parameter values (i.e. those other than Fig. 3 a), the unstable regions are unfeasible.  I dropped the feasibility requirement for the currently implemented code because it made plotting the desired 3spp instability region easier (i.e. without having to deal with 2spp feasible coexistence).  I just need to go through the other figs/models to make sure the same cheap shortcut works for them, too.  Some models/parameter sets, like in Fig 3b below (where there are apparently two unstable regions), for example, might pose problems.

![MoNSTR_Fig3_Full](https://user-images.githubusercontent.com/940249/137401885-c5237acd-42c1-4de0-9a52-e8884cb1f670.png)

**Comment by ingemanKE** [comment]
Trying to delineate where we see unstable coexistence vs extinction seems like a morass to me. Do you agree that we will have sufficiently addressed the reviewer comment by showing regions where stable coexistence breaks down and discussing the implications, if any? Looking at this figure, the addition of unstable/extinction bubbles only reinforces the message that suboptimal defense erodes regions of stable coexistence.

**Comment by marknovak** [comment]
Yes, I think we might get away with not distinguishing them, but will have to word things carefully.  However, given `Manipulate[]`, it might not be hard to distinguish (as long as dynamics within each unstable regions are constant across the full space of each region).
